astropy/modeling/functional_models.py

```diff
@@ -1123,6 +1167,18 @@ def __init__(self, amplitude=amplitude.default, x_0=x_0.default,
             amplitude=amplitude, x_0=x_0, y_0=y_0, r_in=r_in, width=width,
             **kwargs)
 
+    def bounding_box_default(self):
```

Comment:
This is what I had done as part of the `test_bounding_box1D`, `test_bounding_box2D`, and `test_custom_model_bounding_box` functions in `test_models.py`, but I needed custom test parameters and @adonath didn't think it was wise to add these to `example_models.py`. So these tests check for exact agreement within the bounded region but don't check that all of the flux is necessarily captured.